src/bin/cargo-flamegraph.rs

```diff
@@ -309,18 +317,22 @@ fn main() {
     build(&opt);
 
     let workload = workload(&opt);
+    if opt.verbose {
```

Comment:
At least one of the dependencies `inferno`, already relies on `log`, maybe better to use that instead of introducing our own log level machinery.